variables.tf

```diff
@@ -116,6 +116,11 @@ variable "availability_zone_count" {
   type        = number
   default     = 2
   description = "Number of Availability Zones for the domain to use."
+
```

Comment:
This seems to be confusing, the number of availability zones should only be greater than 1 when zone_awareness_enabled is set to true. In our case it is set to false and until this version it was working fine with just one AZ.
With this change we can't use only one AZ (which is supported by the AWS console).
Reading the changelog it states:

_Without this change, it’s possible to set zone_awareness_enabled to true which will automatically use 2 AZs, but set availability_zone_count to 1, which is confusing_

Though I understand the reason, now it is not working for AZ count 1 and zone_awareness_enabled set to **false**

Reply:
@morremeyer @Gowiem 
I guess this change breaks the usage of zone_awareness_enabled set to false with only 1 AZ.